connectors/src/connectors/slack/bot.ts

```diff
@@ -324,12 +324,19 @@ function _processCiteMention(
         .map((key) => {
           const k = key.trim();
           const ref = references[k];
-          if (ref && ref.sourceUrl) {
+          if (ref) {
             if (!refCounter[k]) {
               counter++;
               refCounter[k] = counter;
             }
-            return `[<${ref.sourceUrl}|${refCounter[k]}>]`;
+            const link = ref.sourceUrl
+              ? ref.sourceUrl
+              : `${DUST_API}/w/${
+                  ref.dataSourceWorkspaceId
```

Comment:
Broken links until we backfill -- we're ok with that I presume ?

Reply:
Yes! But I backfill once before I deploy so most will never be broken :+1: